src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/Endpoint/IEndpoint.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public interface IEndpoint
```

Comment:
>IEndpoint [](start = 21, length = 9)

A comment here would really help new future people  that need to add new types of validations.

Reply:
yup, was work in progress